src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -43,7 +43,7 @@
         'list.edit': 'Edit symbol / format'|trans({}, 'Admin.International.Feature'),
         'list.reset': 'Reset'|trans({}, 'Admin.Actions'),
         'list.reset.success': 'Your symbol and format customizations have been successfully reset for this language.'|trans({}, 'Admin.Notifications.Success'),
-        'list.example.format': '%price%'|trans({}, 'Admin.International.Feature'),
+        'list.example.format': '{price}'|trans({}, 'Admin.International.Feature'),
```

Comment:
Same comment (and same for all following files) Is it *required* to change this wording because of how Vue3 now haves? If it's not we should keep the former format